### ISSUE TITLE
Error [ERR_REQUIRE_ESM]: require() of ES Module [...]/node_modules/.pnpm/chalk@5.2.0/node_modules/chalk/source/index.js from [...]/node_modules/.pnpm/browser-sync@2.28.1/node_modules/browser-sync/dist/async.js not supported.

### ISSUE THREAD

**Issue by Anoesj** [BrowserSync/browser-sync#2023]
### Issue details

Ever since the 2.28.1 update, I get this error. Note that I have `"type": "module",` in my `package.json`.
The error message:
```
Error [ERR_REQUIRE_ESM]: require() of ES Module [...]/node_modules/.pnpm/chalk@5.2.0/node_modules/chalk/source/index.js from [...]/node_modules/.pnpm/browser-sync@2.28.1/node_modules/browser-sync/dist/async.js not supported.
Instead change the require of index.js in [...]/node_modules/.pnpm/browser-sync@2.28.1/node_modules/browser-sync/dist/async.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> ([...]/node_modules/.pnpm/browser-sync@2.28.1/node_modules/browser-sync/dist/async.js:7:13)
    at Object.<anonymous> ([...]/node_modules/.pnpm/browser-sync@2.28.1/node_modules/browser-sync/dist/async-tasks.js:1:13)
    at Object.<anonymous> ([...]/node_modules/.pnpm/browser-sync@2.28.1/node_modules/browser-sync/dist/browser-sync.js:3:18)
    at Object.<anonymous> ([...]/node_modules/.pnpm/browser-sync@2.28.1/node_modules/browser-sync/dist/index.js:7:19) {
  code: 'ERR_REQUIRE_ESM'
}
```

### Steps to reproduce/test case

I did not try to create an isolated reproduction, but I expect having `"type": "module",` in a `package.json` should do the trick.

### Please specify which version of Browsersync, node and npm you're running

- Browsersync [2.28.1]
- Node        [18]
- Npm         [pnpm v7.28.0]

### Affected platforms
Note that I only tested this on Ubuntu.
- [x] linux
- [ ] windows
- [ ] OS X
- [ ] freebsd
- [ ] solaris
- [ ] other _(please specify which)_

### Browsersync use-case

- [ ] API
- [x] Gulp
- [ ] Grunt
- [ ] CLI

### If CLI, please paste the entire command below

n/a

### for all other use-cases, (gulp, grunt etc), please show us exactly how you're using Browsersync

```js
browserSync.init({
  server: {
    baseDir: globalThis.paths.src,
    middleware: function (req, res, next) {
      res.setHeader('Access-Control-Allow-Origin', '*');
      next();
    },
  },
  port: 3000,
  single: true,
  https: {
    key: process.env.SSL_LOCALHOST_KEY,
    cert: process.env.SSL_LOCALHOST_CERT,
  },
  open: true,
  ghostMode: false,
  notify: false,
});
```

**Comment by shakyShane** [comment]
@Anoesj - can you try `browser-sync@2.28.2` 🙏🏻

**Comment by Anoesj** [comment]
Unfortunately, still no luck 😕️:
```
Error [ERR_REQUIRE_ESM]: require() of ES Module [...]/node_modules/.pnpm/chalk@5.2.0/node_modules/chalk/source/index.js from [...]/node_modules/.pnpm/eazy-logger@4.0.0/node_modules/eazy-logger/index.js not supported.
Instead change the require of [...]/node_modules/.pnpm/chalk@5.2.0/node_modules/chalk/source/index.js in [...]/node_modules/.pnpm/eazy-logger@4.0.0/node_modules/eazy-logger/index.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> ([...]/node_modules/.pnpm/eazy-logger@4.0.0/node_modules/eazy-logger/index.js:4:13)
    at Object.<anonymous> ([...]/node_modules/.pnpm/browser-sync@2.28.2/node_modules/browser-sync/dist/logger.js:7:14)
    at Object.<anonymous> ([...]/node_modules/.pnpm/browser-sync@2.28.2/node_modules/browser-sync/dist/browser-sync.js:7:14)
    at Object.<anonymous> ([...]/node_modules/.pnpm/browser-sync@2.28.2/node_modules/browser-sync/dist/index.js:7:19) {
  code: 'ERR_REQUIRE_ESM'
}
```

**Comment by Anoesj** [comment]
Fixed by #2026 ✅️
